tools/accuracy_checker/accuracy_checker/postprocessor/corner_crop.py

```diff
@@ -0,0 +1,70 @@
+"""
+Copyright (c) 2018-2020 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from ..config import NumberField, StringField
+from ..utils import get_size_from_config
+from .postprocessor import PostprocessorWithSpecificTargets
+from ..preprocessor import CornerCrop
+from ..representation import ImageProcessingAnnotation, ImageProcessingPrediction
+
+
+class CornerCropImage(PostprocessorWithSpecificTargets):
```

Comment:
if I right remember, we already have simmilar postprocessing:
https://github.com/openvinotoolkit/open_model_zoo/blob/develop/tools/accuracy_checker/accuracy_checker/postprocessor/crop_ground_truth_image.py#L58
can we merge them into 1?